File: metric_tool/test/resources/kotlin/lcam/LcamB.kt
package io.company.app.view.basket.kotlin.lcam

internal class LcamB(private var myField: String) {
    fun setField(value: String) {
        myField = value
    }

    private fun getField(): String {
        return myField
    }

    private fun getSomethingElse(): String {
        return "somethingElse"
    }
}

File: metric_tool/test/resources/dart/lcam/lcam_b.dart
library mylib;

class LcamB {
  LcamB(String myField) : _myField = myField;

  final String _myField;

  String getField() {
    return _myField;
  }

  String getSomethingElse() {
    return "somethingElse";
  }
}
